### PR TITLE
Update put, merge and add ref with schema version

### PR DESCRIPTION
**Pull request by reyreaud-l**
### What's being changed:

Update the following endpoints to have schemaVersion
* PutObject
* MergeObject
* AddReferences
* DeleteObject

### Review checklist

- [ ] Documentation has been updated, if necessary. Link to changed documentation:
- [ ] Chaos pipeline run or not necessary. Link to pipeline:
- [ ] All new code is covered by tests where it is reasonable.
- [ ] Performance tests have been run or not necessary.

### CODE REVIEW

File: usecases/objects/references_update.go [hunk 1]
```diff
@@ -118,20 +118,21 @@ func (req *PutReferenceInput) validate(
...
 
+	// TODO-RAFT: Pull the schemaversion here and return it alongside the class
 	class, err := sm.GetClass(ctx, principal, req.Class)
 	if err != nil {
-		return nil, err
+		return nil, 0, err
 	}
-	return refs, validateReferenceSchema(sm, class, req.Property)
+	return refs, 0, validateReferenceSchema(sm, class, req.Property)
 }
```
Comment (moogacs) [inline usecases/objects/references_update.go:136]: **thought (if-minor):** if it's always gonna be `0` do we need to return version at all?

Comment (moogacs) [reply usecases/objects/references_update.go:136]: i think there could be something missing to return the version 🤔

Comment (moogacs) [reply usecases/objects/references_update.go:136]: same in usecases/objects/references_delete.go

Comment (moogacs) [reply usecases/objects/references_update.go:136]: same in usecases/objects/references_add.go

**Review by moogacs — APPROVED** [review]

File: usecases/objects/references_update.go (hunk 1 above)
Comment (reyreaud-l) [reply usecases/objects/references_update.go:136]: Discussed orally: The idea is to have the placeholder for pulling the version and then we can return it and avoid returning 0 here.

Comment (moogacs) [reply usecases/objects/references_update.go:136]: didn't see the `// TODO-RAFT: Pull the schemaversion here and return it alongside the class`

**Pull request merged**